app/assets/stylesheets/_reports.scss

```diff
@@ -2,10 +2,13 @@
   .container {
     min-height: 100%;
   }
+  .tabs_container{
+  width:73%
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Colon after property should be followed by one space<br>Declaration should be terminated by a semicolon